### DISCUSSION TITLE
Toggleable Fullscreen

### DISCUSSION THREAD

**Discussion by ProgramClass2** [TurboWarp/extensions#774] [Ideas]
How about an extension that lets you manually change if fullscreen is on or off?

![scratchblocks](https://github.com/TurboWarp/extensions/assets/131938567/78c4e73a-8ba1-43e8-a802-e37556951e68)

**Comment by BlueDome77** [comment]
#708

**Reply by ProgramClass2** [reply]
Oh, didn't realize that extension included that option. Thanks!
